app/src/main/java/org/dicekeys/app/encryption/AppKeystore.kt

```diff
@@ -34,6 +30,7 @@ class AppKeystore {
     enum class KeystoreType {
         BIOMETRIC,
         AUTHENTICATION,
+        DEVICE_CREDENTIALS,
```

Comment:
Please document what each of these mean, ideally linking to Android documentation that explains them.  I can't tell if BIOMETRTRIC is biometric-only.  "AUTHENTICATION" is a confusing name given that these are all for authentication, so if it's used because it relates to android functionality of that name it would be valuable to tell the reader that here.

Reply:
Added some comments.

Reply:
Comments are great, but better names would further improve readability.  So that we can get this merged, I'm approving this and proposing the rename in #163.